probe-dictionary/explore.js

```diff
@@ -185,8 +185,8 @@ function showSearchOnlyFilters(show) {
     }
 
     var channel = $("#select_channel").val();
-    if (!["release", "beta", "nightly", "any"].includes(channel)) {
-      $("#select_channel").val("any");
+    if (!["release", "beta", "nightly"].includes(channel)) {
+      $("#select_channel").val("release");
```

Comment:
We should only be defaulting to release when switching to the stats (ie, non-search) view. This looks like it will always switch to release, regardless of tab.

Reply:
@chutten, No, the results are displayed for that channel which is chosen from the drop-down except for `any` for which `release` channel data is shown.
Did I get your right? Does this sound clear? Please correct me if I am wrong. Thanks!

Reply:
It sounds like it might do what I describe, but the only way I can convince myself is if there's a live example I can play with. Can you push this to your github pages so I can play with it? (or is your gh-pages branch already full with something else I've asked for :S )

Reply:
@chutten , Yes, it works the same what the issue asks for. Can you please review #450 . After that, I will be able to make it live. Thanks!